src/sentry/eventstore/base.py

```diff
@@ -146,6 +147,7 @@ def get_events(
         limit (int): Query limit - default 100
         offset (int): Query offset - default 0
         referrer (string): Referrer - default "eventstore.get_events"
+        event_list (Sequence[Event]): List of events to get
```

Comment:
project_id / event_id filtering is the role of the `snuba_filter`. Can we add this argument to the `snuba_filter` instead?